TempleEight/docs/arch/service.md

```diff
@@ -3,3 +3,155 @@ id: service
 title: Service Architecture
 sidebar_label: Service Architecture
 ---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This section will describe the architecture of an individual service.
+We'll use the `ExampleService` from the the [Getting Started](../getting-started.md) guide to illustrate the core service architecture.
+Once covered, we'll look at how optional features such as inter-service communication and metrics slot in.
+
+## Service File Tree
+As a reminder, here is the `ExampleService` block from the [Getting Started](../getting-started.md) guide:
+```
+ExampleService: service {
+  foo: string;
+  bar: int;
+}
+```
+
+And here is the corresponding generated file tree:
```

Comment:
Not a big fan of starting with And

Reply:
Neither, this was my attempt to write `casually`

Reply:
![](https://steamuserimages-a.akamaihd.net/ugc/874119725354054181/086A944A120DD942B321729669E87D3885612BC1/?imw=753&imh=1024&ima=fit&impolicy=Letterbox&imcolor=%23000000&letterbox=true)